extensions/blocks/social-previews/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+
+export const name = 'social-previews';
+
+export const settings = {
+	render: () => <SocialPreviews />,
+};
+
+class SocialPreviews extends Component {
+	render() {
+		return <JetpackPluginSidebar>{ /* TODO: Content will go here! */ }</JetpackPluginSidebar>;
+	}
+}
```

Comment:
Yeah, I think that'd be best to go in that direction in future iterations.

Reply:
maybe we should update the scaffolding logic to generate functions instead of classes

Reply:
> maybe we should update the scaffolding logic to generate functions instead of classes

Yes, that's a good point. I think we had done this in #14686, though?